cmake/stm32/devices.cmake

```diff
@@ -560,18 +581,33 @@ set(STM32_ALL_DEVICES
     G081GB
     G081KB
     G081RB
+    G0B0CE
+    G0B0KE
+    G0B0RE
+    G0B0VE
+    G0B1CB
     G0B1CC
     G0B1CE
+    G0B1KB
     G0B1KC
     G0B1KE
+    G0B1MB
+    G0B1MC
+    G0B1ME
+    G0B1NE
+    G0B1RB
     G0B1RC
     G0B1RE
+    G0B1VB
     G0B1VC
     G0B1VE
     G0C1CC
     G0C1CE
     G0C1KC
     G0C1KE
+    G0C1MC
+    G0C1ME
+    G0C1NE
```

Comment:
```suggestion
```
This one does not exist (today) https://www.st.com/en/microcontrollers-microprocessors/stm32g0x1.html